app/components/thank-you-container.js

```diff
@@ -1,9 +1,103 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed,
+  get,
+  isPresent
 } = Ember;
 
+const MAX_VOLUNTEERS = 12;
+
+/**
+  The `thank-you-container` component presents the main content for the
+  `thank-you` donations page. This includes an icon, thank you message and
+  a list of contributors.
+
+  ## Default Usage
+
+  ```handlebars
+  {{thank-you-container project=project}}
+  ```
+
+  @class thank-you-container
+  @module Component
+  @extends Ember.Component
+ */
 export default Component.extend({
-  classNames: ['thank-you-container']
+  classNames: ['thank-you-container'],
+
+  /**
+    A computed alias for the project's organization members
+
+    @property organizationMemberships
+    @type Ember.Array
+   */
+  organizationMemberships: computed.alias('project.organization.organizationMemberships'),
+
+  /**
+    Filters the array of `organizationMemberships` down to members who are not
+    pending acceptance.
+
+    @property nonPendingMemberships
+    @type Ember.Array
+   */
+  nonPendingMemberships: computed('organizationMemberships.[]', 'organizationMemberships.@each.role', function() {
```

Comment:
@JoshSmith I had an idea of why I was doing this, but I wasn't sure I was right. So I threw together this [twiddle to show the differences](https://ember-twiddle.com/5596716e1499fb3cb86c77806222800e?openFiles=templates.components.computed-each-attr.hbs%2Ctemplates.components.computed-each-attr.hbs).

- computed property on an array of objects won't recompute on things being pushed/popped from the array if `obj.[]` is not used.
- computed property on an array of objects won't recompute if an object in that arrays attributes change without `@each.thatattribute`

However, I agree with your statement below, so this will probably be taken out. But I figured I should still investigate.